compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/AMDAIETileAndFuse.cpp

```diff
@@ -85,10 +85,11 @@ std::optional<scf::SCFFuseProducerOfSliceResult> tileAndFuseProducerOfSlice(
   // 2a. Compute the destination operands to use for the cloned operation.
   SmallVector<Value> origDestinationTensors, clonedOpDestinationTensors;
   Operation *fusableProducerOp = fusableProducer.getOwner();
-  if (isa<DestinationStyleOpInterface>(fusableProducerOp) &&
-      failed(tensor::getOrCreateDestinations(
-          rewriter, fusableProducerOp->getLoc(), fusableProducerOp,
-          origDestinationTensors)))
+  if (isa<tensor::PackOp>(fusableProducerOp) ||
```

Comment:
Nit: Can we just split this condition. Mixed `||` and `&&` conditions arent really readable.